gitlab/payload.go

```diff
@@ -288,7 +288,7 @@ type Project struct {
 	Description       string `json:"description"`
 	WebURL            string `json:"web_url"`
 	AvatarURL         string `json:"avatar_url"`
-	GitSSSHURL        string `json:"git_ssh_url"`
+	GitSSHURL         string `json:"git_ssh_url"`
```

Comment:
Good shot! @wasphin but what do you think about backward compatibility? May be this field active in use already?

Reply:
I am new to golang, maybe we can declare the exactly ref we used? Or maybe we can select another branch to apply non-compatible changes?

Reply:
I think it's a questions to have new v6 branch... I'll think about it, thanks anyway!